tests/sims/feegrant/operations_test.go

```diff
@@ -86,6 +86,8 @@ func (suite *SimTestSuite) getTestingAccounts(r *rand.Rand, n int) []simtypes.Ac
 
 	// add coins to the accounts
 	for _, account := range accounts {
+		acc := suite.accountKeeper.NewAccountWithAddress(suite.ctx, account.Address)
+		suite.accountKeeper.SetAccount(suite.ctx, acc)
```

Comment:
All tests in the test suite are affected, now they pass. Before that they all failed.  

Account is properly added to the keeper because all the tests passes. Why I should check that when the production code will failed if it is not. If we follow that logic we should check that account is fund correctly or that the context is properly created and so on.